src/main/java/com/github/wovnio/wovnjava/WovnServletFilter.java

```diff
@@ -30,6 +30,11 @@ public void init(FilterConfig config) throws ServletException {
     @Override
     public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws ServletException, IOException
     {
+        if (RequestOptions.wovnDisableMode((HttpServletRequest)request)) {
```

Comment:
Not particularly a fan of making this static, even if it doesn't happen to use settings for that method. You construct an instance of RequestOptions further down in this method, so you can just move that up..

Reply:
This checks happens in the main `doFilter` method, while I requestOptions is created and used in `tryTranslate` after deciding to process the request. I figured I wanted to do as little as possible before checking if we want to touch the request at all.

But I suppose I could instantiate it here and pass it down. It's what we do with `headers`, after all.

Reply:
done